datalad/distribution/create_publication_target_sshwebserver.py

```diff
@@ -249,29 +230,27 @@ def __call__(sshurl, target=None, target_dir=None,
                     else:
                         raise ValueError("Do not know how to hand existing=%s" % repr(existing))
 
-                cmd = ssh_cmd + ["mkdir", "-p", path]
                 try:
-                    runner.run(cmd)
+                    ssh(["mkdir", "-p", path])
                 except CommandError as e:
                     lgr.error("Remotely creating target directory failed at "
                               "%s.\nError: %s" % (path, str(e)))
                     continue
 
             # init git repo
-            cmd = ssh_cmd + ["git", "-C", path, "init"]
+            cmd = ["git", "-C", path, "init"]
             if shared:
                 cmd.append("--shared=%s" % shared)
             try:
-                runner.run(cmd)
+                ssh(cmd)
             except CommandError as e:
                 lgr.error("Remotely initializing git repository failed at %s."
                           "\nError: %s\nSkipping ..." % (path, str(e)))
                 continue
 
             # check git version on remote end:
-            cmd = ssh_cmd + ["git", "version"]
             try:
-                out, err = runner.run(cmd)
+                out, err = ssh(["git", "version"])
                 git_version = out.lstrip("git version").strip()
```

Comment:
lstrip takes set of characters to strip... it would work somewhat as far as version doesn't contain any characters from 'git version' string, but imho even 'out.strip().split()[-1]' could be a cleaner (although would need some guarding/check for -1 indexing)